inginious/frontend/i18n/es/LC_MESSAGES/messages.po

```diff
@@ -1407,15 +1694,15 @@ msgstr "Agregar"
 
 #: inginious/frontend/plugins/grader_generator/pages/templates/grader.html:162
 msgid "Entity (only on VHDL)"
-msgstr ""
+msgstr "Entidad (solo en VHDL)"
 
 #: inginious/frontend/plugins/grader_generator/pages/templates/grader.html:173
 msgid "Testbench"
-msgstr ""
+msgstr "Banco de pruebas"
```

Comment:
Incorrect translation. This is not necessary to translate as it name in English is more understandable.